README.md

```diff
@@ -1 +1,46 @@
-# Future Debian/Ubuntu package for The Lounge
+# Debian/Ubuntu package for The Lounge
+This repository holds out the build scripts that generates our .deb precompiled packages and also tracks Debian-specific issues in relation to the packaging.
+
+## Installation
+If you are looking to simply install The Lounge, please use our pre-compiled binary .deb files available in the releases section of the main project. This section assumes you want to build a Debian package from sources.
+
+```
+# Clone the repository
+$ git clone https://github.com/thelounge/arch-lounge.git
+$ cd deb-lounge
+
+# Call the build script
+$ ./build-package
+```
+
+After this, you should have a nice `.deb` file in the `deb/` output folder! This file can then be installed:
+
+```
+# dpkg -i deb/*.deb
+```
+
+### Configuration
+The default system-wide documentation file is located at `/etc/thelounge/config.js`. Please note that user profiles and their IRC passwords are also stored there (moving to `/var/lib/thelounge` is planned but not yet done), so the directory is only readable by the `thelounge` user.
+
+### Running
+The Lounge provides both a system-wide and per-user systemd unit. If you installed the package, The Lounge should already be running and accessible on `http://127.0.0.1:9000`.
```

Comment:
Copy that.
As long as it doesn't imply trade-offs on the system-wide, or hold us in any way, I don't see why not keeping it. But if having 2 ever becomes a problem (maintenance, confusion, I don't know, just wild guesses), I understand the system-wide way to be the official one and the user-wide way to be a bonus.

Actually, my main worry is that administrators with little experience (hey, there is nothing wrong is being a noob, and being a noob willing to self-host is a noble cause 😄) may find confusing to have to choose without fully knowing why.
What do you think of changing the title to `System (recommended)`?